guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,56 +1,67 @@
-For example, if you make a {RHEL} 8.3 repository with a few excluded packages, `dnf update` could fail.
-Instead of turning on dependency solving to resolve the issue, investigate the error from DNF and adjust the filters to stop excluding the missing dependency.
-If dependency solving is enabled, the repository may stray from {RHEL} 8.3 as discussed above.
-endif::[]
+[id="Combining_exclusion_filters_with_dependency_solving"]
+.Combining exclusion filters with dependency solving
+====
+You want to recreate {RHEL}{nbsp}8.3 using Content View filters and include selected errata from a later {RHEL}{nbsp}8 minor release.
+To achieve this, you create filters to exclude most of the errata after the {RHEL}{nbsp}8.3 release date, except a few that you need.
+Then, you enable dependency solving.
 
-If a Content View version is out of date, packages that are part of a newer erratum may depend on packages that are not within the Content View version.
-Before incrementally adding the erratum with dependency solving, consider publishing a new Content View version instead.
-Incrementally adding the erratum with dependency solving might pull in many new packages, which could alter the intended scope of the Content View version.
+In this situation, dependency solving might include more packages than expected.
+As a result, the client digresses from being a {RHEL}{nbsp}8.3 machine.
 
-If a single package is incrementally added to a Content View version, dependency solving may be needed to install that package.
-Keep in mind that the dependency solving recommendations above also apply to incremental updates.
+If you do not need the extra errata and packages, do not configure Content View filtering.
+Instead, enable and use the {RHEL}{nbsp}8.3 repository on the *Content* > *Red Hat Repositories* page in the {ProjectWebUI}.
+====
 
-If dependency solving must be used, consider using more Content Views with fewer repositories and combining them together using composite Content Views.
-Dependency solving time increases significantly with more repositories because every repository in a Content View is examined for dependencies.
-
-.Enabling Dependency Solving
-You can select to resolve package dependencies per Content View or have it enabled by default in *Settings > Content > "content view Dependency Solving Default"*.
-{Team} does not recommend enabling Content View dependency solving by default unless the pitfalls above are well-understood.
+ifeval::["{client-os-family}" == "Red Hat"]
+In some cases, excluding packages makes dependency solving impossible for DNF.
+For example, if you make a {RHEL}{nbsp}8.3 repository with a few excluded packages, `dnf update` can fail.
+Instead of enabling dependency solving to resolve the problem, investigate the error from `dnf` and adjust the filters to stop excluding the missing dependency.  Otherwise, dependency solving might cause the repository to digress from {RHEL} 8.3.
+endif::[]
```

Comment:
Could this be rewritten to not include a specific RHEL version?

Reply:
I could do this:

`For example, if you make a repository based on a minor release of Red Hat Enterprise Linux with a few excluded packages, dnf update can fail. Instead of enabling dependency solving to resolve the problem, investigate the error from dnf and adjust the filters to stop excluding the missing dependency. Otherwise, dependency solving might cause the repository to digress from the minor Red Hat Enterprise Linux release.`

But it doesn't read as well as when a specific example version is used. I think it's okay to use RHEL 8.3 because the whole thing is introduced as an example, isn't it? But I don't have a strong opinion on the matter, so if you think the above version without 8.3 is better, I'll do it that way.

Reply:
I didn't see **example** written explicitly, so if you somehow add that to what you already have that is fine. Or maybe include it in the title?

Reply:
There are two examples that specify RHEL 8.3.

* line 49 uses an example block; the word "example" doesn't appear in asciidoc source, only in the rendered output
* line 65 starts with "For example"

Is it possible you missed the *example* part in one of these? If so, maybe I should think about rewriting that example to make it clearer.